src/gsi/write_incr.f90

```diff
@@ -493,8 +494,8 @@ subroutine write_fv3_inc_ (grd,sp_a,filename,mype_out,gfs_bundle,ibin)
           if (should_zero_increments_for('rwmr_inc')) qrsm(:,:,k) = 0.0_r_kind
           out3d(:,:,krev) = transpose(qrsm(j1:j2,:,k))
        end do
-       call nccheck_incr(nf90_put_var(ncid_out, qrvarid, sngl(out3d), &
-                         start = ncstart, count = nccount))
+       !call nccheck_incr(nf90_put_var(ncid_out, qrvarid, sngl(out3d), &
```

Comment:
We could comment out the entire `if (iqr>0) then` block instead of only the call to `nccheck_incr`